tools/flang2/flang2exe/CMakeLists.txt

```diff
@@ -92,7 +92,7 @@ set(INCLUDE_DIRS
   ${FLANG_SOURCE_DIR}/lib/scutil
   ${CMAKE_CURRENT_SOURCE_DIR}
   ${CMAKE_CURRENT_BINARY_DIR}
-  ${CMAKE_CURRENT_SOURCE_DIR}/${TARGET_ARCHITECTURE}-${TARGET_OS}
+  ${CMAKE_CURRENT_SOURCE_DIR}/${TARGET_ARCHITECTURE}-Linux
```

Comment:
Could you remind me why we need this change?

Reply:
There's only one existing target architecture.

Reply:
https://github.com/flang-compiler/flang/pull/726 fixes this